NEWS.md

```diff
@@ -47,6 +68,11 @@ The feed template for the URLHaus feed contained a spelling error:
 The correct name for the parameter "delimeter" is "delimiter". Please fix your configured bots.
 The `intelmqctl upgrade-config` command automatically fixes a configuration if the misspelling is detected.
 
+#### IntelMQ-Manager -> IntelMQ-API
+`IntelMQ-Manager` PHP backend is getting replaced by newly developed `IntelMQ-API`. The `IntelMQ-Manager` release only contains the files for web frontend. Install the `IntelMQ-API`, in order to upgrade your local installation. Follow the instructions in our [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html) to install `IntelMQ-API` & setup authentication. Unfortunately you can't keep current authentication users, you have to add users as described in the [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html#adding-a-user).
```

Comment:
**the** newly developed `IntelMQ-API`... for **the** web frontend...

And I would it phrase like this: The `intelmq-manager` packages now depend or recommend the installation of the `intelmq-api` package, therefore a normal upgrade should pull in the API. Follow the instructions in our documentation to configure the API. It uses its own authentication backend now, how to setup users is described in the documentation.